src/commcare_cloud/commcare_cloud.py

```diff
@@ -112,22 +112,21 @@
 def run_on_control_instead(args, argv, force_latest_code):
     executable = 'commcare-cloud'
     branch = getattr(args, 'branch', 'master')
-    default_virtualenv = "ansible" if sys.version_info[0] == 2 else "cchq"
-    venv = os.environ.get("CCHQ_VIRTUALENV", default_virtualenv)
+    venv = os.environ.get("CCHQ_VIRTUALENV")
     if force_latest_code:
         bash_commands_template = (
             '{env}cd ~/commcare-cloud && git fetch --prune && git checkout {branch} '
             '&& git reset --hard origin/{branch} && source ~/init-ansible && {cchq} {cchq_args}'
         )
     else:
         bash_commands_template = (
-            '{env}cd ~/commcare-cloud && source ~/.virtualenvs/${{CCHQ_VIRTUALENV:-cchq}}/bin/activate '
+            '{env}source ~/commcare-cloud/control/activate_venv.sh --fail-on-error '
```

Comment:
I think this causes a "discontinuity problem" when executing `cchq --control ...` _from_ a system with this change (this PR) _against_ a control environment that does not have this change (yet).  In that scenario, commands are executed against the remote control system that depend on this file (`activate_venv.sh`) being present, but since it isn't, the command crashes.

Reply:
Thanks for flagging this Joel. My initial thought is that a change like this should have had a changelog/commcare-cloud announcement with it as well. I'll put that together now.

Reply:
See https://github.com/dimagi/commcare-cloud/pull/5649